imagepicker/src/main/java/com/esafirm/imagepicker/features/camera/CameraModule.kt

```diff
@@ -9,6 +9,7 @@ typealias OnImageReadyListener = (List<Image>?) -> Unit
 
 interface CameraModule {
     fun getCameraIntent(context: Context, config: BaseConfig): Intent?
+    fun getVideoIntent(context: Context, config: BaseConfig): Intent?
```

Comment:
As mentioned above "camcorder" or "video camera" is more suitable.

So this would be `getCamcorderIntent` or `getVideoCameraIntent`